client/starwhale/core/model/cli.py

```diff
@@ -572,6 +574,10 @@ def _run(
         swcli model run --workdir . --module mnist.evaluator --handler 1
         # run index fullname(mnist.evaluator:MNISTInference.cmp) handler from the working directory, search mnist.evaluator module to get runnable handlers
         swcli model run --workdir . --module mnist.evaluator --handler mnist.evaluator:MNISTInference.cmp
+
+        \b
+        # --> run with the args defined in your handler
+        swcli model run --workdir . --module mnist.evaluator --handler 1 -- --a=1 --b 3 --a 4 --d dataset/version/latest
```

Comment:
not a good example, for cli, when one char will use `-{c}` in common.

```
swcli model run --workdir . --module mnist.evaluator --handler 1 -- --external-a=1 --external-b 3 -c 4 -d dataset/version/latest
```

Reply:
done